Platforms/QemuSbsaPkg/VirtNorFlashStandaloneMm/VirtNorFlashStandaloneMm.c

```diff
@@ -0,0 +1,356 @@
+/** @file  NorFlashStandaloneMm.c
+
+  Copyright (c) 2011 - 2021, Arm Limited. All rights reserved.<BR>

```

Comment:
Same for this file, I guess? Not sure how much was derived from the dxe version?

Reply:
Updated.